wormhole-connect/src/components/Button.tsx

```diff
@@ -28,11 +28,15 @@ const useStyles = makeStyles()((theme: any) => ({
     backgroundColor: theme.palette.button.action,
     color: theme.palette.button.actionText,
   },
+  text: {
+    backgroundColor: 'transparent',
+  },
 }));
 
 type Props = {
   action?: boolean;
   elevated?: boolean;
+  text?: boolean;
```

Comment:
I used the same name they used in [material UI for the buttons](https://mui.com/material-ui/react-button/#basic-button). But I understand that it can get confusing, I could put something like `textStyle`, `link`, or `withoutBackground`.